lib/libc/minimal/include/ctype.h

```diff
@@ -23,6 +23,11 @@ static inline int isalpha(int c)
 	return (int)((((unsigned)c|32u)-(unsigned)'a') < 26U);
 }
 
+static inline int isblank(int c)
+{
+	return (int)((c == (int)' ') || (c == (int)'\t'));
+}
+
 static inline int isspace(int c)
```

Comment:
I'd let the compiler do something cute if it likes, but express these in a direct fashion.
```
static inline int isspace (int c)
{
    return c == ' ' || ('\t' <= c && c <= '\r');
}
```
Similarly for the other functions. There's no reason to obscure what these functions do; having to check that there are 5 'blank' values between `'\t'` and `'\r'` is something the compiler should do, not us.

Reply:
I'll prepend a commit for that, since it's technically not related to the bugfix.